packages/tocco-ui/src/Panel/PanelBody.js

```diff
@@ -4,43 +4,66 @@ import React from 'react'
 import StyledPanelBody from './StyledPanelBody'
 
 /**
- * Only <PanelBody/> is affected by the visibility state.
+ * Only <Panel.Body/> is affected by the visibility state.
  */
 class PanelBody extends React.Component {
+  observer = null
```

Comment:
i dont think this initialization with null is necessary?

same applies vor the state?